guide/daily-spending-wallet/requesting.md

```diff
@@ -34,439 +34,421 @@ Illustration source
+   modalImage = "/assets/images/guide/daily-spending-wallet/requesting/PaymentLink@2x.png"
    layout = "shadow"
-   caption = "Sharing a payment request contactlessly is a quick way to request a payment in-person."
-   alt-text = "Screen showing contactless UI."
+   caption = "Payment links make it simple for senders to import a payment request into their app."
+   alt-text = "Screen showing a payment link being opened by a wallet."
    width = 250
    height = 541
+   modalWidth = 250
+   modalHeight = 541
 %}
 
 </div>
 
-## Receiving a payment
-
-Once a request has been shared, the sender then needs to send bitcoin to the request. We cover the design considerations and user flows for sending bitcoin in our [sending bitcoin]({{ "/guide/payments/send/" | relative_url }}) page.
+## Payment addresses
 
-### Lightning
+Payment addresses, also known as [reusable payment requests]({{ "/guide/how-it-works/payment-request-formats/#invoice-protocols" | relative_url }}), are requests that can receive more than a single payment. These are not widely supported but offer unique use cases, features, and improved usability. 
 
-Receiving a payment is something that involves a lot of background complexity. How a wallet application manages this, has a major impact on the end user's experience.
+Due to the unique use cases of payment addresses, its recommended having a separate area within your app to create, manage, and share these.
 
-To receive a payment, the receiver needs a payment channel with inbound liquidity and also has to be online. Some wallets offer services that allow channels to be opened on-demand if a user has no inbound liquidity when receiving an incoming payment. They may also hold payments for a user until they are online to forward it to them. These services require a wallet to be solely connected to the peer that offers these services. We cover what services are commonly offered and how they work on our Lightning services page.
-
-Without Lightning services, a user will need to obtain some inbound liquidity before receiving a payment. This can be done by opening a channel with a peer that offers inbound liquidity and/or sending payments moving outbound to inbound capacity. It is recommended to let users know if they do not have any inbound liquidity before they share a payment request to prevent a payment failure.
+An example of how a payment address could be used is as quickly shared request from a users profile, say to receive tips on their website. If you are using payment addresses you should let users know that other wallets may not support sending to them and that the user needs to be online to receive payments.
 
 <div class="image-slide-gallery">
 
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/funding/opening-channel.png"
-   retina = "/assets/images/guide/daily-spending-wallet/funding/opening-channel@2x.png"
+   image = "/assets/images/guide/daily-spending-wallet/requesting/PaymentAddress.png"
```

Comment:
Should there be an extra item in this list for managing payment addresses?

Reply:
Wasn't sure if we should keep that for a dedicated page on this and just have this as a stub for them for now until its more widely adopted

Reply:
Agree, lets come back to this